driver/CBAInferFusion_driver.hpp

```diff
@@ -601,7 +601,7 @@ int CBAInferFusionDriver<Tgpu, Tref>::AllocateBuffersAndCopy()
     else
         out_sz = in_sz; // This is for N+A so the output is the same as the input size
 
-    if(miopen::IsEnabled(ENV(MIOPEN_DRIVER_PAD_BUFFERS_2M)))
+    if(miopen::IsEnabled(MIOPEN_ENV(MIOPEN_DRIVER_PAD_BUFFERS_2M)))
```

Comment:
This must be reverted and fixed in CK

Reply:
I agree that CK is at fault for exposing `ENV` but it doesn't hurt us to add our prefix to our macros. I'll leave it up to @bghimireamd to see if he wants to go through with changing a large set of files.